functions.php

```diff
@@ -57,6 +64,54 @@ public function __construct() {
 	}
 
 	/**
+	 * Show copyright text field.
+	 *
+	 * @param array $args
+	 */
+	public function copyright_show_settings( $args ) {
+		$copyright = get_option( 'copyright', '' );
+
+		printf(
+			'<input type="text" name="copyright" class="regular-text" value="%1$s" id="%2$s" />',
+			esc_attr( $copyright ),
+			esc_attr( $args['label_for'] )
+		);
+	}
+
+	/**
+	 * Function to add copyright text block in general options
+	 */
+	public function add_copyright_text() {
+		add_settings_section(
+			'copyrighttext_id',
+			'',
+			'',
+			'general'
+		);
+
+	/**
+	 * Register taxonomies for page.
+	 */
```

Comment:
Minor - Single line comment inside function body.

Reply:
Hi @dashaluna . I am confused here. Before this I already made a single line comment but after reviewed by you 
https://github.com/greenpeace/planet4-master-theme/pull/7#discussion-diff-141847991R105
I made this change.
Can you please confirm

Reply:
@sagarsdeshmukh , Comments right before the function should be multiline. 
Comments inside a function should be single line. 
your "Register taxonomies for page" is inside the function.

Reply:
Thanks @koyan :)

@sagarsdeshmukh coding standard page on codex is good to refer to when in doubt 
https://codex.wordpress.org/WordPress_Coding_Standards
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/

I still keep looking at it now and again, as there is a lot of info.